rtengine/settings.h

```diff
@@ -95,6 +96,13 @@ class Settings
     double          cbdlsensi;
 //    bool            showtooltip;
     bool            itcwb_enable;
+    bool            itcwb_custom_enable;
+    double          itcwb_deltaspec;
+    double          itcwb_tempstdA;
+    int             itcwb_maxsize;
+    double          itcwb_Ypurple;
+    double          itcwb_powponder;
```

Comment:
These are no longer used, correct?

Reply:
I keep itcwb_enable, itcwb_deltaspec, itcwb_powponder because they can be used
itcwb_enable - allows custom use of Itcwb instead of just auto. Indispensable for certain situations, in particular "Low sampling & no use camera settings" (see Rawpedia)

itcwb_deltaspec - allows the user to see the characteristics of the patch in "verbose" mode (see Rawpedia)

itcwb_powponder : keep to modify behavior of algorithm